README.md

```diff
@@ -44,6 +44,12 @@ __Check if it's all good with the draft.__ To see your draft live locally, run J
 
 __Publish.__ If it's all good and you're ready to publish it, change the date on the file and move it to `_posts`.
 
+### Setting Social Media info
+
+On default the first 50 words will be used as a preview in social media. You can overwrite this by setting  `description: you description` in the meta section.
+
+You can specify a image that is used for social media previews by setting
+`images: path_to/image.img` in the meta section.
```

Comment:
is it really called meta section?

Reply:
no, the real name is "front matter" which nobody understands :)

Reply:
i would still be in favor of using the official name, or at least mention it in addition.